.eslintrc.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
     'require-jsdoc': 'off',
     'camelcase': 'off',
     'new-cap': ['error', {capIsNew: false}],
+    'no-unexpected-multiline': 'off'
```

Comment:
Needed this because originally eslint did not like having the `[...]` function calls for the Typechain bindings in the L2LPTGateway tests on a separate line.